### ISSUE TITLE
Add Frictionless Data package metadata to published Darwin Core archives

### ISSUE THREAD

**Issue by MattBlissett** [gbif/ipt#1502]
A Zip archive can contain both Darwin Core metadata and Frictionless Data metadata.  There's even a tool to [do this automatically](https://frictionlessdata.io/blog/2020/01/22/frictionless-darwincore/).

**Comment by mike-podolskiy90** [comment]
@MattBlissett Is it something we still need?

**Comment by MattBlissett** [comment]
I don't think it needs to be part of the milestone, but it's worth keeping the issue open.

In the end, it might be more useful on downloads from GBIF.org.
